ProjectSource/src/test/java/com/kayako/sdk/auth/FingerprintAuthPojoTest.java

```diff
@@ -0,0 +1,31 @@
+package com.kayako.sdk.auth;
+
+import com.aurea.unittest.commons.pojo.Testers;
+import com.aurea.unittest.commons.pojo.chain.TestChain;
+import com.openpojo.reflection.impl.PojoClassFactory;
+import com.openpojo.validation.Validator;
+import org.junit.Test;
+
+import javax.annotation.Generated;
+
+@Generated("GeneralPatterns")
+public class FingerprintAuthPojoTest {
+
+    @Test
+    public void validateFingerprintAuthGetters() {
+        Validator validator = TestChain.startWith(Testers.getterTester()).buildValidator();
+        validator.validate(PojoClassFactory.getPojoClass(FingerprintAuth.class));
```

Comment:
⚠️  In this case, it could be a good improvement to avoid the creation of this variable.  This way the code becomes shorter and "garbage variables" are avoided.  
          
## The possible solution in this case

```JAVA
TestChain.startWith(Testers.getterTester())
    .buildValidator()
    .validate(PojoClassFactory.getPojoClass(FingerprintAuth.class)); 
```
 

<a id="e8ba5310-50c3-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />